src/Form/FieldInput.js

```diff
@@ -181,6 +195,10 @@ FieldInput.propTypes = {
   // Optional. Which field property is currently being edited
   // (usually passed down from form definition)
   editing: React.PropTypes.string,
+  // Optional. Specify if the field should be focused
+  // Useful in combination with preset values, will set cursor to end of input
+  // (usually passed down from form definition)
+  focused: React.PropTypes.bool,
```

Comment:
Are you sure? What if there are multiple input fields? Which one will be focused then (probably the last rendered one)? 
The idea behind this property is to declare **one** field that should be focused when the form is rendered.